php_phongo.c

```diff
@@ -982,6 +982,8 @@ void php_phongo_read_preference_to_zval(zval *retval, const mongoc_read_prefs_t
 	} else {
 		ADD_ASSOC_NULL_EX(retval, "tags");
 	}
+
+	ADD_ASSOC_LONG_EX(retval, "maxStalenessMS", mongoc_read_prefs_get_max_staleness_ms(read_prefs));
```

Comment:
I don't think this should be added if maxstaleness is the default (0), just like we're no longer to do that for "journal", "w", etc.